CONTRIBUTING.md

```diff
@@ -301,6 +301,35 @@ A PR (pull request) is considered to be **ready to merge** when:
 Any [maintainer](./README.md#contributing) can merge the PR once it is **ready
 to merge**.
 
+## Reviewer guidelines
+
+Semantic conventions consist of multiple [areas](./AREAS.md) with ownership
+defined in the [CODEOWNERS](./.github/CODEOWNERS) file.
+
+When a PR is raised against specific area(s), it is recommended to allow the corresponding
+area(s) owners to review and iterate on it first before approving or rejecting the PR.
+
+A review from [@specs-semconv-approvers](https://github.com/orgs/open-telemetry/teams/specs-semconv-approvers)
+is required on every PR and, in most cases, follows after area(s) owners approval.
+
+Before merging a PR, [@specs-semconv-maintainers](https://github.com/orgs/open-telemetry/teams/specs-semconv-maintainers)
+MUST verify that the PR has been approved by the corresponding code owner(s). For
```

Comment:
```suggestion
MUST verify that the PR has been approved by the corresponding area owner(s). For
```
To distinguish this from the global approvers as defined in https://github.com/open-telemetry/semantic-conventions/blob/6da8f44c5c227b8a09f2f78eada1656ace27727a/.github/CODEOWNERS#L15-L16?